### ISSUE TITLE
Readability

### ISSUE THREAD

**Issue by jthaman** [alphapapa/emacs-package-dev-handbook#21]
Two issues:

The colors are bad. I cannot read the code. Just make it black on white. 

The TOC is big and obscures some of the page.

**Comment by alphapapa** [comment]
Thanks for your feedback.

**Comment by olnw** [comment]
I agree about the colours. Personally, the contrast between the text and background is not high enough, which makes it slightly difficult to read.

**Comment by alphapapa** [comment]
I guess Solarized isn't as popular as it used to be.

**Comment by olnw** [comment]
Maybe — I could just be accustomed to the Modus themes. However, I think some brighter colours could be used, mainly in the code blocks.

**Comment by alphapapa** [comment]
I guess we need modus-vivendi for HTML then.  :)
